sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -1174,34 +1170,34 @@ def delete_blob(
 
     @overload
     def download_blob(
```

Comment:
azure\storage\blob\_container_client.py:1172: error: Overloaded function signatures 1 and 2 overlap with incompatible return types  [misc]

Same as in blob_client